build/Targets/References.targets

```diff
@@ -66,11 +66,11 @@
 
         <!-- NuGet Dependencies -->
         <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
-        <PackageReference Include="System.Reflection.Metadata" Version="1.4.1-beta-24430-01" />
+        <PackageReference Include="System.Reflection.Metadata" Version="1.4.2" />
         <PackageReference Include="System.ValueTuple" Version="4.3.0" />
         <PackageReference Include="Microsoft.Composition" Version="1.0.27" />
         <PackageReference Include="Newtonsoft.Json" Version="8.0.3" />
-        <PackageReference Include="Microsoft.VisualStudio.LanguageServices" Version="2.0.0-rc2" />
+        <PackageReference Include="Microsoft.VisualStudio.LanguageServices" Version="2.3.0-beta1-61703-10" />
```

Comment:
No, this is necessary because I did some work in Roslyn to expose the constructors for the `CommandLineSourceFile` and `CommandLineReference` structs and this is the first build that contains these changes.  This won't build otherwise.